pkg/printer/printer_test.go

```diff
@@ -1090,3 +1090,157 @@ Jan  1 00:20:34.567   k8s1   cilium-test/pod-to-a-denied-cnp-75cb89dfd-vqhd9 (ID
 		})
 	}
 }
+
+func TestPrinter_WriteServerStatusResponse(t *testing.T) {
+	buf := bytes.Buffer{}
+	ss := &observerpb.ServerStatusResponse{
+		NumFlows:  2031,
+		MaxFlows:  4095,
+		SeenFlows: 2348885,
+		UptimeNs:  301515181665,
+		Version:   "cilium v1.10.3+g4145278",
+	}
+	ssn := &observerpb.ServerStatusResponse{
+		NumFlows:            2771,
+		MaxFlows:            8190,
+		SeenFlows:           2771,
+		UptimeNs:            301515181665,
+		Version:             "hubble-relay v1.10.3+g4145278",
+		NumConnectedNodes:   &wrapperspb.UInt32Value{Value: 2},
+		NumUnavailableNodes: &wrapperspb.UInt32Value{Value: 0},
+	}
+	type args struct {
+		ss *observerpb.ServerStatusResponse
+	}
```

Comment:
kind of an odd contraption. looks like there may have been more args in there at one point

Reply:
Yeah, I just re-used the style of the existing tests. I wouldn't have used an `args` struct as there's only one argument anyways.